modules/RTC/JitsiLocalTrack.js

```diff
@@ -559,13 +559,18 @@ export default class JitsiLocalTrack extends JitsiTrack {
      * NOTE: used only for audio tracks to detect audio issues.
      */
     _setByteSent(tpc, bytesSent) {
-        this._bytesSent = bytesSent;
+        this._bytesSentPerPC.set(tpc.id, bytesSent);
         const iceConnectionState = tpc.getConnectionState();
 
         if (this._testByteSent && iceConnectionState === 'connected') {
             setTimeout(() => {
-                if (this._bytesSent <= 0) {
-                    // FIXME: Remove ${this}
```

Comment:
is this fix still something we want to do? if so maybe we should fix it or keep the comment

Reply:
While I was debugging the issue, printing `this` was quite useful. That's why I removed the comment.